### PR TITLE
Lab2 Polina Minina

### PR DESCRIPTION
**Pull request by pminina01**

### CODE REVIEW

File: app_python/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ENV PROJECT_DIR "/app"
+
+RUN mkdir -p $PROJECT_DIR
+WORKDIR $PROJECT_DIR
+COPY . $PROJECT_DIR
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+EXPOSE 5000
+CMD [ "flask", "run", "--host=0.0.0.0", "--port=5000" ]
```
Comment (sevenzing) [inline app_python/Dockerfile]: I think you forget to add **Run as non-root user** practice

Comment (sevenzing) [reply app_python/Dockerfile]: also, runnning in "prod" mode with `flask run` is not good idea, look at `gunicorn`

**Review by sevenzing — APPROVED** [review]

File: DOCKER.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* Do consolidation for consecutive instructions
+* Store share data in volumes (as config file for example)
+* Set Log Rotation
+* Use linters for Dockerfile
+
+## Linters choice
+
+* For docker: [hadolint](https://github.com/hadolint/hadolint)
+    ```bash
+    docker run --rm -i hadolint/hadolint < Dockerfele
```
Comment (bragov4ik) [inline DOCKER.md]:
```suggestion
    docker run --rm -i hadolint/hadolint < Dockerfile
```

**Review by bragov4ik — APPROVED** [review]

File: app_go/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FROM golang:1.16-alpine
+
+ENV PROJECT_DIR "/app"
+RUN mkdir -p $PROJECT_DIR
+WORKDIR $PROJECT_DIR
+COPY . $PROJECT_DIR
+
+EXPOSE 3000
+CMD [ "go", "run", "app.go" ]
```
Comment (Ilya-Kolomin) [inline app_go/Dockerfile:9]: Better practice to implement multi-stage build here.

**Review by Ilya-Kolomin — APPROVED** [review]
Review OK👌

**Pull request closed without merging**